mod-mosaic/schemas/mosaic_order.json

```diff
@@ -138,19 +130,26 @@
       "type": "string",
       "$ref": "../../mod-orders-storage/schemas/workflow_status.json"
     },
-    "materialSupplier": {
-      "description": "UUID for the material supplier",
-      "$ref": "../../common/schemas/uuid.json"
-    },
-    "accessProvider": {
-      "description": "UUID for the access provider",
-      "$ref": "../../common/schemas/uuid.json"
-    },
     "format": {
       "description": "Order format (Physical Resource, Electronic Resource, P/E mix, Other)",
       "type": "string",
       "$ref": "../../mod-orders-storage/schemas/order_format.json"
     },
+    "physical": {
+      "description": "details of this purchase order line relating to physical materials",
+      "type": "object",
+      "$ref": "../../mod-orders-storage/schemas/physical.json"
+    },
+    "eresource": {
+      "description": "eresource-related details of this purchase order line",
+      "type": "object",
+      "$ref": "../../mod-orders-storage/schemas/eresource.json"
+    },
+    "checkinItems": {
+      "description": "if true this will toggle the Check-in workflow for details associated with this PO line",
```

Comment:
better to add to description what means true and what means false because it always confusing